changelogs/unreleased/5363-shadialtarsha-minor.md

```diff
@@ -0,0 +1,27 @@
+## Default Global RateLimit Policy
+
+This Change adds the ability to define a default global rate limit policy in the Contour configuration 
+to be used as a global rate limit policy by all HTTPProxy objects.
```

Comment:
worth mentioning it can be disabled if not desired on an httpproxy

Reply:
Will add it